client/landing/jetpack-cloud/section.js

```diff
@@ -1,6 +1,6 @@
 export const JETPACK_CLOUD_SECTION_DEFINITION = {
 	name: 'jetpack-cloud',
-	paths: [ '/jetpack-cloud' ],
+	paths: [ '/', '/scan', '/backups' ],
```

Comment:
Can we define this inside client/sections.js instead?

Reply:
Wouldn't `paths: [ '/' ]` alone suffice? Why do we have to explicitly define `/scan` and `/backups`?

Reply:
> Can we define this inside client/sections.js instead?

Yes, but we're consciously not doing it right now because we're creating Jetpack Cloud only with this section to keep it clean. In the future, we could be combining those if we decide that we want to expose Calypso routes for Jetpack Cloud. So let's keep it as-is for this first version and iterate later.

Reply:
> Wouldn't `paths: [ '/' ]` alone suffice? Why do we have to explicitly define `/scan` and `/backups`?

No, they won't. This would work if we had the entire Jetpack Cloud in a subroute, but because it is the landing route itself, then it needs all the first level routes to be defined so Express.js would know about them and would handle them properly.